src/domUpdates.js

```diff
@@ -1,23 +1,26 @@
 // Imports
-import {recipeData} from './data/recipes'
-import {recipeGrid} from './scripts'
+import { 
+   recipeGrid,
+   tagArea 
+  } from './scripts';
+import { recipeData } from './data/recipes';
 
 // functions
 
 const makeRecipeColumnData = (data) => {
   const mappedRecipe = data.map((recipe, index) => {
     return {
-      column: index+1,
+      column: (index+1) % 3,
       id: recipe.id,
       image: recipe.image,
       name: recipe.name,
       tags: recipe.tags
     }
   });
 
-  const leftColumn = mappedRecipe.filter(recipe => recipe.column % 3 === 1);
-  const centreColumn = mappedRecipe.filter(recipe => recipe.column % 3 === 2);
-  const rightColumn = mappedRecipe.filter(recipe => recipe.column % 3 === 0);
+  const leftColumn = mappedRecipe.filter(recipe => recipe.column === 1);
+  const centreColumn = mappedRecipe.filter(recipe => recipe.column === 2);
+  const rightColumn = mappedRecipe.filter(recipe => recipe.column === 0);
```

Comment:
nice change here. Love that

Reply:
Hey thank you! I felt it was not too readable earlier.